db/migrate/20160411143918_add_organisation_id_to_enrollments.rb

```diff
@@ -1,6 +1,6 @@
 class AddOrganisationIdToEnrollments < ActiveRecord::Migration
   def change
     add_column :flood_risk_engine_enrollments, :organisation_id, :integer, index: true
-    add_foreign_key :flood_risk_engine_enrollments, :flood_risk_engine_organisations
+    add_foreign_key :flood_risk_engine_enrollments, :flood_risk_engine_organisations, column: :organisation_id
   end
```

Comment:
@autotelik I don't think column is required if rails can derive it because - I think rails convention will step in cos the  column is :organisation_id ?